swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/Cacheable.kt

```diff
@@ -39,7 +39,7 @@ open class InMemoryCache<out T : Cacheable>(
 
   override fun contains(key: String?): Boolean {
     if (key == null) return false
-    return cache.get().find { it.name == key } != null
+    return get().find { it.name == key } != null
```

Comment:
Are you sure?

Reply:
Pretty sure. This makes it use the `get()` function below, so that if the cache is null it will refresh the cache then return the cache. Found this out while testing.